code/modules/html_interface/html_interface.dm

```diff
@@ -116,12 +116,12 @@ mob/verb/test()
 	src.height         = height
 	src.head           = head
 
-/datum/html_interface/Del()
+/datum/html_interface/Destroy()
 	src.closeAll()
 
 	html_interfaces.Remove(src)
 
-	return ..()
```

Comment:
The `/datum/proc/Destroy()` is a DIRECT hard del, it would fuck up pooling.

blame @eswordthecat